### DISCUSSION TITLE
xterm retina screens

### DISCUSSION THREAD

**Discussion by yvess** [XQuartz/XQuartz#61] [Q&A] [answered]
Hi

I'm trying to figuring out settings to get crips sharp fonts in xterm for a retina screen (2x scaling).
I tried different Xft.dpi settings in .Xresources but without luck the fonts just get blown up.
Googling also don't lead me to a good solution.
Also tried xrandr but without luck (screen went black). 

It would be great if this than also works for forwarded X11 Apps. I also often run a a remote Sublime Text Editor from a 
Linux host. 

Has somebody settings that generate nice clean/sharp fonts in XQuartz?
I'm using an MacBook Air M1 attached to a LG Ultrafine 27" 5k. 
 
Content Like this would be great for wiki a page. 

thanks
Yves

PS: Awesome that there is already an Apple Silicon build! Thanks a lot

**Comment by zhiwei-dong** [comment]
Also meet this problem, edit org.macosforge.xquartz.X11 do not work. When using `xrandr --size 3840x2160` can make shaper, but will force get into full-screen mode.

**Comment by jeremyhu** [comment, marked as answer]
Yeah, unfortunatley, nobody has put much effort into updating XQuartz for retina (#91), so it is still a bit of a sub-par experience.

The proper fix would be to replace the existing XQuartz DDX with the Xorg DDX and use a compositing window manager to display window content inside real AppKit windows.  Work on this started about 7 years ago by some students as a senior year project, but we never got it to the point of shipping in the product.  I'm hoping that this effort can be revitalized and also used as the basis for a wayland compositing window manager at the same time.
